enterprise/cypher/cypher/pom.xml

```diff
@@ -164,7 +164,7 @@
     <dependency>
       <groupId>org.neo4j</groupId>
       <artifactId>neo4j-cypher-compiler-2.3</artifactId>
-      <version>2.3.10</version>
+      <version>2.3.11</version>
```

Comment:
why not declare them in one place in a root pom and only import in all those poms instead of changing all of them always?

Reply:
We like your thinking @MishaDemianenko but will do that as a separate PR ( @sherfert has made a card at our wall for it)